### PR TITLE
fix: style tweaks and bug fixes

### PR DESCRIPTION
**Pull request by cisel**
# Description

- Style fixes for both billboard and navigation dropdown to fix bugs in fs-core 
- Cloning the children part changed due to changed incoming contentful data

Screenshot 2021-09-27 at 10 11 20
Screenshot 2021-09-27 at 10 11 32
Screenshot 2021-09-27 at 10 11 59
Screenshot 2021-09-27 at 10 12 16

Screenshot 2021-09-27 at 10 14 12
Screenshot 2021-09-27 at 10 14 05
Screenshot 2021-09-27 at 10 14 22
Screenshot 2021-09-27 at 10 16 45

# Checklist

Pull request contains:

- [ ] A new component
- [x] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [x] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [x] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

File: src/compounds/navigation-dropdown/src/stories.tsx [hunk 1]
```diff
@@ -12,35 +12,21 @@ export default {
...
-    </li>
-  </ul>
-)
+const navConfig = {
+  'http://localhost:3000/current-accounts/': 'Please select',
+  'http://localhost:3000/current-accounts/cashback/': 'Cashback',
+  'http://localhost:3000/current-accounts/high-interest/': 'High Interest',
+  'http://localhost:3000/current-accounts/most-popular/': 'Most popular',
+  'http://localhost:3000/current-accounts/packaged/': 'Packaged',
+  'http://localhost:3000/current-accounts/standard/': 'Standard'
```
Comment (samhstn) [inline src/compounds/navigation-dropdown/src/stories.tsx:21]: Do you want to keep these as https://www.uswitch.com/current-accounts?tracking=false
We shouldn't need fs-core running for trustyle to work

**Review by samhstn — COMMENTED** [review]
What happens if we just write these normally as `jsx` instead of `createElement`?

File: src/compounds/navigation-dropdown/src/stories.tsx (hunk 1 above)
Comment (cisel) [reply src/compounds/navigation-dropdown/src/stories.tsx]: Right, nice one! I've made it to work faster but then it escaped my attention. Thanks! it's updated now.

**Comment by cisel** [comment]
I've updated the `createElement`. I had turned `cloneElement` into `createElement` while trying to solve the issue but it's more readable now. Thanks for the comment.

**Review by samhstn — APPROVED** [review]

**Review by erkinun — APPROVED** [review]

**Pull request merged**